db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java

```diff
@@ -664,13 +661,17 @@ void newResourceInput(String resourceName) {
 	 * used at each carriage return to show that it is still "live"
 	 * when it is reading multi-line input.
 	 */
-	static void doPrompt(boolean newStatement, LocalizedOutput out, String tag) 
+	 static void doPrompt(boolean newStatement, LocalizedOutput out, String tag)
 	 {
+	 	final SimpleDateFormat sdf = new SimpleDateFormat("yyyy.MM.dd HH:mm:ss");
```

Comment:
Please use ISO 8601 delimiters for the date: yyyy-MM-dd

Reply:
Good point! thanks, I will fix it.